corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -10,51 +10,75 @@
+                class="btn btn-default"
+                data-bind="
+                    attr: {'data-target': '#lookup-table-prompt-' +   property.uniqueId}"
+                data-toggle="modal">
+          <i class="fa fa-pencil"></i>
+          {% trans "Configure" %}
+        </button>
+      </td>
       {% endif %}
-      {% if js_options.search_prompt_appearance_enabled %}
-        <td class="col-sm-2">
-          <select class="form-control" data-bind="value: property.appearance">
-            <option value="">{% trans "Text" %}</option>
-            <option value="barcode_scan">{% trans "Barcode" %}</option>
-            <option value="fixture">{% trans "Lookup Table Selection" %}</option>
-          </select>
-          <button type="button"
-                  class="btn btn-default"
-                  data-bind="
-                        visible: property.appearance() == 'fixture',
-                        attr: {'data-target': '#lookup-table-prompt-' + property.uniqueId}"
-                  data-toggle="modal">
-            <i class="fa fa-pencil"></i>
-            {% trans "Edit" %}
-          </button>
-          <span data-bind="visible: property.itemSet.nodeset, text: property.itemSet.nodeset"></span>
-          <div class="modal fade" data-bind="attr: {'id': 'lookup-table-prompt-' + property.uniqueId}">
-            <div class="modal-dialog modal-md">
-              <div class="modal-content">
-                <div class="modal-header">
-                  <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
-                  <h4 class="modal-title">
-                    {% trans "Lookup Table:" %}
-                    <!-- ko text: property.label --><!-- /ko -->
-                  </h4>
-                </div>
-                <div class="modal-body">
-                  <div class="form-horizontal">
+      <td class="col-sm-2">
+        <div class="modal fade" data-bind="attr: {'id': 'lookup-table-prompt-' + property.uniqueId}">
+          <div class="modal-dialog modal-md">
+            <div class="modal-content">
+              <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                <h4 class="modal-title">
+                  {% trans "Options for:" %}
+                  <!-- ko text: property.label --><!-- /ko -->
+                </h4>
+              </div>
+              <div class="modal-body">
+                <div class="form-horizontal">
+                  <fieldset>
+                    <legend>{% trans "Basic" %}</legend>
+                    <div class="form-group">
+                      <label class="control-label col-xs-12 col-sm-3">
+                        {% trans "Help Text" %}
+                      </label>
+                      <div class="col-xs-12 col-sm-9">
+                        <input class="form-control" type="text" data-bind="value: property.hint"/>
```

Comment:
UX detail: this would be a good place for a textarea, I bet these are going to get long.

Reply:
Sure